lib/logstash/environment.rb

```diff
@@ -51,47 +51,21 @@ def test?
       env.downcase == "test"
     end
 
-    # set GEM_PATH for logstash runtime
-    # GEM_PATH should include the logstash gems, the plugin gems and the bootstrap gems.
-    # the bootstrap gems are required specificly for bundler which is a runtime dependency
-    # of some plugins dependedant gems.
     def set_gem_paths!
-      ENV["GEM_PATH"] = ENV["BUNDLE_PATH"] = logstash_gem_home
-      ENV["BUNDLE_GEMFILE"] = GEMFILE_PATH unless ENV['BUNDLE_GEMFILE']
-
-      require 'bundler'
-      require 'logstash/bundler_patch'
-
-      Bundler.setup
-      ENV["GEM_HOME"] = ENV["GEM_PATH"] = logstash_gem_home
-
-      # Bundler.setup will wipe the existing $LOAD_PATH.
-      # Since we are using gems not declared in the gemfile we need to
-      # recalculate the $LOAD_PATH with all the available gems.
-      load_paths = Gem::Specification
-        .collect(&:load_paths)
-        .flatten
-        .reject { |path| $LOAD_PATH.include?(path) }
-
-      $LOAD_PATH.unshift(*load_paths)
-    end
-
-    def bundler_install_command(gem_file, gem_path)
-      # for now avoid multiple jobs, ex.: --jobs 4
-      # it produces erratic exceptions and hangs (with Bundler 1.7.9)
-      options = [
-        "install",
-        "--gemfile=#{gem_file}",
-        "--path",
-        gem_path,
-      ]
+      require 'rubygems'
+
+      begin
```

Comment:
I'd say the method name should be changed. I wanted to do it but never got to it.  probably `bundler_setup` ?